userspace/libsinsp/parsers.cpp

```diff
@@ -4499,6 +4499,46 @@ void sinsp_parser::parse_setgid_exit(sinsp_evt *evt)
 	}
 }
 
+static std::string convert_to_string(Json::ValueType other) {
+	std::string value_type("Not a valid json type");
+	switch(other) {
+	case Json::ValueType::nullValue:
+		return "nullValue";
+	case Json::ValueType::intValue:
+		return "intValue";
+	case Json::ValueType::uintValue:
+		return "uintValue";
+	case Json::ValueType::realValue:
+		return "realValue";
+	case Json::ValueType::stringValue:
+		return "stringValue";
+	case Json::ValueType::booleanValue:
+		return "booleanValue";
+	case Json::ValueType::arrayValue:
+		return "arrayValue";
+	case Json::ValueType::objectValue:
+		return "objectValue";
+	default:
+		break;
+	}
+	return value_type;
+}
+
+bool sinsp_parser::check_is_convertible(const Json::Value& value, Json::ValueType other, std::string field, bool log_message)
```

Comment:
`const std::string& field`? Or maybe even `const char*` to avoid creating temporary objects during every parse?

Reply:
done